Addons/tsgExoticQuadrature.hpp

```diff
@@ -130,29 +151,32 @@ inline double lagrange_eval(size_t idx, const std::vector<double> &roots, double
  * (weight_fn_vals[i] + shift) for every i.
  * \endinternal
  */
-inline TasGrid::CustomTabulated getShiftedExoticQuadrature(const int n, const double shift,
+inline TasGrid::CustomTabulated getShiftedExoticQuadrature(const int n, const double shift, const std::vector<double> &weight_fn_vals,
```

Comment:
Why do we need `weight_fn_vals` here. Are those used outside of the `assert()` statement?

I thought that at this stage, we already have the weight function merged into the `shifted_weights`.

Reply:
I looked over the code and it looks like we don't need it after all. I removed it in the latest push.